pitch.md

```diff
@@ -1,60 +1,112 @@
+seems like it will use the "Production Rule" model heavily as it involves a lot of
+specifying what a command is and what it should do. I'm sure I'll realize more as
+the project goes on, but right now the only semantic error I can foresee is if the
+user specified the same command multiple times. This can be reported to the user
+on first parse, giving the user the chance to rename the command.
 
 ## Expressiveness
 _What should be easy to do in this language? What should be possible, but
 difficult? What should be impossible or very difficult?_
 
+Specifying what commands can be said in chat to interact with the game and
+how those commands should be processed should be very easy. Pretty much
+anything else should be hard.
 
 ## Related work
 _Are there any other DSLs in this domain? If not, describe how you know there
 aren't and conjecture why not. If so, describe them and provide links. How well 
 do they address the need? Are there any particularly admirable qualities of the
 language? Are there parts of the language you think could be improved?_
 
+There are plenty of Twitch bots that allow chat to interact with the stream, 
+such as by suggesting what music should be played. A popular one is
+[Moobot](http://twitch.moobot.tv/). There is also a lot of work by the
+Tool Assisted Speedrun (TAS) community on scripting (
+[usually in Lua](http://tasvideos.org/LuaScripting.html)) to interact
+with games. However, as far as I know, there hasn't been work to bridge the
+two.
 
 ## Suitability
 _What percentage of your time do you think will be spent directly engaging in
 the **language** aspects of this project (e.g., making language design
 decisions), as opposed to "systems" aspects of the project (e.g., implementing a
 complicated semantics that doesn't require a lot of language design)?_
 
+I think time would be split more towards systems at the beginning, when making an
+MVP. But once we progress past single command=single input, I have been extremely
+conflicted about how to deal with any kind of global state or conditionals, which
+can be very important in some games.
 
 ## Scope
 _How big an idea is this? How ambitious is this project?_
 
+This is a project that should be relatively easy to get something working, but new
+features could be added and the language grow to whatever size I want.
 
 ## Challenges and opportunities
 _Why is this a good project? What are some challenges you expect to face? How
 might you overcome them? What are some warning signs that the project has gotten
 off track, and how will you get the project back on track if needed?_
+
+This is a good idea as I am excited about it, there are a couple of interesting
+language decisions I would enjoy grappling with, and I think it could be very
+beneficial to people who are interested in crowdsourcing a game. If I start spending
+too much time looking at the Twitch API and how to interface with an emulator, I would
+need to reevaluate progress on the project.
```

Comment:
This seems like a really cool idea and, from what you've said, a relatively untapped domain.  Also, it seems to have a reasonable scope that you can build off of.  Good luck!